python/_tskitmodule.c

```diff
@@ -6982,11 +6982,17 @@ TableCollection_load(TableCollection *self, PyObject *args, PyObject *kwds)
     PyObject *ret = NULL;
     PyObject *py_file;
     FILE *file = NULL;
-    static char *kwlist[] = { "file", NULL };
+    tsk_flags_t options = 0;
+    int skip_tables = false;
+    static char *kwlist[] = { "file", "skip_tables", NULL };
 
-    if (!PyArg_ParseTupleAndKeywords(args, kwds, "O", kwlist, &py_file)) {
+    if (!PyArg_ParseTupleAndKeywords(
+            args, kwds, "O|i", kwlist, &py_file, &skip_tables)) {
         goto out;
     }
+    if (skip_tables) {
+        options |= TSK_LOAD_SKIP_TABLES;
```

Comment:
We need to add a test to the ``tests/test_lowlevel.py`` to cover this. You could add a test like ``load_skip_tables`` [here](https://github.com/tskit-dev/tskit/blob/ab99547d024c737698ec8ce6f06a2d8d56e52302/python/tests/test_lowlevel.py#L166)  that does some very simple tests on the ``skip_tables`` argument (do we get a TypeError on bad types) and just checks that it runs ok with ``skip_tables=True``. (No need to test any properties of the ts, we'll do that elsewhere)

Will need to do the same thing for the TreeSequence.load method. This can be a copy-paste basically into the appropriate test class.